cookies/operations.py

```diff
@@ -389,3 +430,58 @@ def generate_collection_coauthor_graph(collection,
         nx.set_node_attributes(graph, 'uri', node_uris)
 
     return graph
+
+@mock.patch('requests.get')
+def concept_search(text, mock_get):
+    """
+    Edit :class`.Entity` instance by linking a :class:`.ConceptEntity` URI with
+    it. This provides functionality to search for a URI based on the text
+    entered in the search field.
+
+    BlackGoat API is used to search for the URIs and the sources by querying
+    with the text entered.
+
+    Parameters
+    ----------
+    text : Search text for the URIs
```

Comment:
And the name ``text`` is a bit misleading, since "text" in Amphora usually evokes the notion of a document. Perhaps ``query``?